### PR TITLE
 test(stocks): add valid and invalid stock symbol tests, closes #372

### LINKED ISSUE #372: Add Cypress tests for Stocks
Add Cypress tests for the [Stocks example](https://tomastrajan.github.io/angular-ngrx-material-starter#/examples/stock-market).

Leave a comment if you would like to take this issue.

### PR DESCRIPTION
**Pull request by huyen1990**
## What:

Issue number: N/A

### CODE REVIEW

File: cypress/integration/stocks.spec.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+describe('STOCKS Actions', () => {
+  beforeEach(() => {
+    cy.visit('/#/examples/stock-market');
+  });
+
+  it('should display data for a valid stock symbol', () => {
+    const goodStockSymbol = 'NVDA';
+
+    cy.get('input[placeholder="Stock symbol"]').type(
+      '{selectall}' + goodStockSymbol
```
Comment (timdeschryver) [inline cypress/integration/stocks.spec.ts]: Nit, what about using string interpolation here?
```suggestion
      `{selectall}${goodStockSymbol}`
```

Comment (timdeschryver) [inline cypress/integration/stocks.spec.ts]: Nit, what about using string interpolation here?
```suggestion
      `{selectall}${goodStockSymbol}`
```

File: cypress/integration/stocks.spec.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    );
+
+    cy.get('.mat-card-title').should('contain', goodStockSymbol);
+  });
+
+  it('should display not found error for an invalid stock symbol', () => {
+    const badStockSymbol = 'BADSTOCK';
+
+    cy.get('input[placeholder="Stock symbol"]').type(
+      '{selectall}' + badStockSymbol
```
Comment (timdeschryver) [inline cypress/integration/stocks.spec.ts]: Same here.

**Comment by huyen1990** [comment]
Is it OK now? git comment good also?

**Comment by tomastrajan** [comment]
LGTM!

@huyen1990 the log says you closed it `huyen1990 closed this 11 hours ago` but now is open so its alright.

**Pull request merged**